astropy/time/tests/test_basic.py

```diff
@@ -867,51 +867,62 @@ def test_explicit_example(self):
         assert t.to_value('mjd', subfmt='str') == '54321.000000000001'
         assert t.to_value('mjd', 'bytes') == b'54321.000000000001'
         expected_long = np.longdouble(54321.) + np.longdouble(1e-12)
-        assert t.to_value('mjd', subfmt='long') == expected_long
+        # Check we're the same to within the double holding jd2
+        # (which is less precise than longdouble on arm64).
+        assert np.allclose(t.to_value('mjd', subfmt='long'),
+                           expected_long, rtol=0, atol=2.**-52)
```

Comment:
I haven't been involved in the other discussions, so `2**-52` looks like a magic number to me. And given that it appears so many times, maybe it should be assigned to a global variable for reuse and a comment to explain why this value?

Reply:
Good point, will do.